tests/unit/color_picker.coffee

```diff
@@ -48,3 +48,16 @@ test 'Color picker converts color to hex when color is invalid', ->
   testHexConversion "foo", undefined
   testHexConversion "rgb(a, b, c)", undefined
   testHexConversion "rgb(1,2,3)", undefined
+
+test 'Custom selected color is set as custom color', ->
+  color = '#addec0'
+  colorPicker.set 'selectedColor', color
+  equal colorPicker.get('customColor'), color
+
+test 'Clicking color picker cell sets selectedColor', ->
```

Comment:
Not sure why this test isn't passing. The click doesn't seem to be working but the selector is definitely present. Took a look with @Dylnuge and we couldn't figure out what was wrong. Any ideas @igillis @cyril-sf?